### PR TITLE
bugs fixing, small changes in a frontend, added simple_form wrappers …

### PR DESCRIPTION
**Pull request by Kasvit**
…for forms

### CODE REVIEW

File: app/assets/stylesheets/dashboard.scss [hunk 1]
```diff
@@ -1,32 +1,8 @@
...
-  color: white;
-  border-color: black;
-  border-width: 1px;
-}
\ No newline at end of file
+// .form-inputs{
+//   max-width: 80%;
+// }
+
+// lable {
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/dashboard.scss]: lable  ?

do you need this? if not - remove garbage

File: app/views/shared/_navbar.html.slim [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
           | My projects
-      / .navbar-item.has-dropdown.is-hoverable
-        a.navbar-link
-          |  My Tasks 
-        .navbar-dropdown
-          a.navbar-item
-            |  List 
-          a.navbar-item
-            |  Calendar 
+        / .navbar-item.has-dropdown.is-hoverable
```
Comment (igorkasyanchuk) [inline app/views/shared/_navbar.html.slim:17]: it's OK to uncomment it and keep, just add href='#'

File: config/initializers/simple_form.rb [hunk 3]
```diff
@@ -8,11 +8,26 @@
...
+    b.use :input, class: "input"
+    b.use :hint,  wrap_with: { tag: :div, class: "help" }
+    b.use :error, wrap_with: { tag: :div, class: "help is-danger" }
+  end
+  
+  config.wrappers :inline_checkbox, :tag => 'div', :class => 'control-group', :error_class => 'error' do |b|
+    b.use :html5
+    b.wrapper :tag => 'div', :class => 'controls' do |ba|
+      ba.use :label_input, :wrap_with => { :class => 'checkbox inline' }
+      ba.use :error, :wrap_with => { :tag => 'span', :class => 'help-inline' }
```
Comment (igorkasyanchuk) [inline config/initializers/simple_form.rb]: use new hash style

File: app/views/shared/_flash.html.slim [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.is-marginless
+  - if flash.notice
+    .notification.is-success
+      = notice
+      button.delete aria-label="delete"
```
Comment (igorkasyanchuk) [inline app/views/shared/_flash.html.slim]: click on "close" works?

**Comment by AndriySolonyna** [comment]
Please add screenshot with UI

**Comment by Kasvit** [comment]
image
image
image

**Comment by igorkasyanchuk** [comment]
Sign up form is too wide. reduce the width and center it

File: app/assets/stylesheets/dashboard.scss (hunk 1 above)
Comment (Kasvit) [reply app/assets/stylesheets/dashboard.scss]: done

File: app/views/shared/_navbar.html.slim (hunk 2 above)
Comment (Kasvit) [reply app/views/shared/_navbar.html.slim:17]: done

File: config/initializers/simple_form.rb (hunk 3 above)
Comment (Kasvit) [reply config/initializers/simple_form.rb]: done

File: app/views/shared/_flash.html.slim (hunk 4 above)
Comment (Kasvit) [reply app/views/shared/_flash.html.slim]: yes
done

File: app/assets/stylesheets/dashboard.scss (hunk 1 above)
Comment (igorkasyanchuk) [reply app/assets/stylesheets/dashboard.scss]: what "done"? I see this file is still here

File: app/views/account/projects/show.html.slim [hunk 5]
```diff
@@ -1,4 +1,12 @@
-.conteiner.box
+.conteiner
```
Comment (igorkasyanchuk) [inline app/views/account/projects/show.html.slim]: conteiner ????

File: config/initializers/simple_form.rb (hunk 3 above)
Comment (igorkasyanchuk) [reply config/initializers/simple_form.rb]: again ... what done?

**Comment by Kasvit** [comment]
image
image

File: app/assets/javascripts/dashboard.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+document.addEventListener('DOMContentLoaded', () => {
+  // Get all "navbar-burger" elements
+  const $navbarBurgers = Array.prototype.slice.call(document.querySelectorAll('.navbar-burger'), 0);
```
Comment (romaslmd) [inline app/assets/javascripts/dashboard.js]: don't use jquery-like variable names for javascript elements
could be very confusing

File: app/assets/javascripts/dashboard.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  // Check if there are any navbar burgers
+  if ($navbarBurgers.length > 0) {
+    // Add a click event on each of them
+    $navbarBurgers.forEach( el => {
+      el.addEventListener('click', () => {
+        // Get the target from the "data-target" attribute
+        const target = el.dataset.target;
+        const $target = document.getElementById(target);
```
Comment (romaslmd) [inline app/assets/javascripts/dashboard.js]: data target is a selector, so use of `getElementById` would be incorrect for class/tag selectors
better to use `document.querySelector(target)`

Comment (Kasvit) [reply app/assets/javascripts/dashboard.js]: If I use querySelector I have got an error: "Uncaught TypeError: Cannot read property 'classList' of null  at HTMLAnchorElement.el.addEventListener"

**Comment by Kasvit** [comment]
image

**Comment by Kasvit** [comment]
image

File: app/assets/stylesheets/dashboard.scss [hunk 8]
```diff
@@ -1,32 +1,8 @@
-.navbar{
-  background-color: #f7f5f5;
-  border-bottom: 1px solid #ececec;
-  margin-bottom: 15px;
-  padding-bottom: 5px;
-
-  .asana_logo {
-    height: auto;
-    align-items: center;
+.self_checkbox {
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/dashboard.scss]: move to bulma_ext.scss

and apply to all checkboxes

File: app/views/account/projects/index.html.slim [hunk 9]
```diff
@@ -1,15 +1,17 @@
...
-.conteiner.box
+.container 
   .level
     .level-left
-      .level-item
-        = link_to "Add project", new_account_project_path, class: "button  is-link"
-  .conteiner
+      .title My projects
+    .level-right
+      = link_to "Add project", new_account_project_path, class: "button  is-link"
```
Comment (igorkasyanchuk) [inline app/views/account/projects/index.html.slim]: remove additional space

File: app/views/account/tasks/_form.html.slim [hunk 10]
```diff
@@ -1,9 +1,10 @@
...
-    = f.input :title, placeholder: 'Title'
-    = f.input :description, placeholder: 'Description'
-    .lavel.lavle-left
-      .lavel-item
-        = f.submit
-      .lavel-item
-        = link_to 'Back', account_project_path(@project)
+    .field
+      = f.input :title, placeholder: 'Title', wrapper: :bulma_input
+      = f.input :description, placeholder: 'Description', wrapper: :bulma_input
```
Comment (igorkasyanchuk) [inline app/views/account/tasks/_form.html.slim]: .field
   = input

.field
   = input

File: app/views/devise/registrations/new.html.slim [hunk 11]
```diff
@@ -1,24 +1,21 @@
...
-                hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
-                input_html: { placeholder: "New Password" },
-                label: false
-      = f.input :password_confirmation, required: true,
-                input_html: { placeholder: "Cofirm Password" },
-                label: false
-    .form-actions
-      = f.submit 'Sign up', class: 'button is-primary'
-= render 'devise/shared/links'
+.hero-body
```
Comment (igorkasyanchuk) [inline app/views/devise/registrations/new.html.slim:1]: where is .hero?

File: app/views/devise/sessions/new.html.slim [hunk 12]
```diff
@@ -1,19 +1,13 @@
...
-      = f.input :password,
-                required: false,
-                input_html: { plaseholder: "Password" },
-                label: false
-      = f.input :remember_me, as: :boolean if devise_mapping.rememberable?
-    .form-actions
-      = f.submit "Log in", class: 'button is-pimary'
-
-= render "devise/shared/links"
+.hero-body
```
Comment (igorkasyanchuk) [inline app/views/devise/sessions/new.html.slim:1]: the same

File: config/initializers/simple_form.rb [hunk 13]
```diff
@@ -8,11 +8,26 @@
 #
 # Use this setup block to configure all options available in SimpleForm.
 SimpleForm.setup do |config|
-  # Wrappers are used by the form builder to generate a
-  # complete input. You can remove any component from the
-  # wrapper, change the order or even add your own to the
-  # stack. The options given below are used to wrap the
-  # whole input.
+
+  config.wrappers :bulma_input, class: "field", error_class: :field_with_errors do |b|
```
Comment (igorkasyanchuk) [inline config/initializers/simple_form.rb:12]: verify how form looks when fields has errors

**Comment by Kasvit** [comment]
image
image
image

**Review by AndriySolonyna — APPROVED** [review]

**Pull request merged**